build/lib/reader/reader.js

```diff
@@ -51,7 +52,10 @@ var Reader = /** @class */ (function () {
                     case 0: return [4 /*yield*/, utils_1.getType(this.file)];
                     case 1:
                         fileType = _b.sent();
-                        if (!(fileType != undefined)) return [3 /*break*/, 6];
+                        if (fileType == undefined)
```

Comment:
this isn't maybe the best way to check for undefined variables, see https://stackoverflow.com/questions/3390396/how-to-check-for-undefined-in-javascript